template.yaml

```diff
@@ -47,6 +47,12 @@ Parameters:
     Default: 2
     Description: Number of vCPUs for the compute node instance type
 
+  ComputeNodeMemory:
```

Comment:
Why statically define this? Why not auto-detect RealMemory and have users provide a SchedulableMemory percentage (see [ParallelCluster for example](https://docs.aws.amazon.com/parallelcluster/latest/ug/slurm-mem-based-scheduling-v3.html#slurm-mem-based-scheduling-realmemory-v3))?

Reply:
`SchedulableMemory` 'll be something that needs further research and for my next PR. :)

IFAIK, `SchedulableMemory` is a Slurm configuration that's shipped with AWS ParallelCluster. And we are not dealing with parallelcluster here. As for the auto-detect for `RealMemory`; we need an ec2 describe call for the instance specified. This will also need more broader change which is beyond the scope of this PR.